content/en/blog/2019/introducing-istio-operator/index.md

```diff
@@ -126,7 +127,7 @@ $ kubectl edit istiocontrolplane example-istiocontrolplane -n istio-system
 To upgrade to a new version of Istio, run:
 
 {{< text bash >}}
-$ kubectl apply -f https://<repo URL>/<new version>/operator-profile-default.yaml
+$ kubectl apply -f https://<repo URL>/<new version>/default-cr.yaml
```

Comment:
@ostromart Same confusion about the version in the path name here? What is different about upgrade vs just applying or editing the CR to change the config?